modules/test/playwright/tests/notification-web/main/emailNotificationTemplate.spec.ts

```diff
@@ -34,6 +37,11 @@ const notificationTemplateInfo = {
 };
 
 test.beforeEach(async ({apiHelpers}) => {
+	notificationTemplate =
```

Comment:
If only your test uses this, then you should not create it before each test. Move this to the test body and right after it add the following:

```
	apiHelpers.data.push({
		id: notificationTemplate.id,
		type: 'notificationTemplate',
	});
```

This will automatically clean it up.